package.json

```diff
@@ -6,8 +6,9 @@
     "build:drive": "npm run build:drive:browser",
     "build:drive:browser": "npm run internal:build -- --env.app=drive --env.target=browser --env.production",
     "build:drive:mobile": "npm run internal:build -- --env.app=drive --env.target=mobile --env.production",
-    "build:photos": "npm run build:photos:browser",
+    "build:photos": "npm run build:photos:browser && npm run build:photos:services",
```

Comment:
`npm-run-all build:photos:browser build:photos:services`

and if we need/want , you can use `--parallel`

Reply:
Ok, I actually copy-pasted from cozy-banks :)
Is there a reason we wouldn't want `--parallel`?

Reply:
For your use case, nope